tests/testthat/test-name_usage.r

```diff
@@ -232,3 +232,42 @@ test_that("fails with more than 1", {
     "length\\(rank\\) == 1 is not TRUE")
 })
 
+
+# paging
+test_that("paging: class data and meta not modified by paging", {
```

Comment:
You are right. It takes too much time. 
Unfortunately it's impossible to make it shorter because gbif takes chunks of 1000 and takes time for doing it.
I have commented all tests about paging (a6742f9). I added the suggestion to uncomment them after test caching will be implemented via `vcr` package.